syntaxes/css-hubl.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "css-hubl",
+  "scopeName": "text.css.hubl",
+  "comment": "HubL CSS Templates",
+  "patterns": [
+    {
+      "include": "source.css"
+    }
```

Comment:
Is there a reason we're not using`{ "include": "source.hubl" }` here as a second pattern like we are in `html-hubl` and are instead using the `injectionSelector`?

Reply:
Yeah, the HTML syntax plays a lot nicer with the HubL syntax (which I believe is because of the way tags get tokenized). CSS, on the other hand, doesn't and so HubL needs to be injected.

Injection should work for HTML, I'll test that out real quick to see if we can clean that up.

Reply:
Can you elaborate on what doesn't work?